poc/README.md

```diff
@@ -0,0 +1,14 @@
+# Proof of Concepts
+
+The scripts in this folder are only intended as PoC cases. Their goal is to
+only verify the architecture of DSS on different platforms.
+
+## How to run
+
```

Comment:
We can mention that you need sudo privileges to run the script.

Reply:
done